config/pepperConfig.js.ctmpl

```diff
@@ -9,6 +9,8 @@ var DDP_ENV = {
     doLocalRegistration: false,
     {{ with secret (printf "secret/pepper/%s/%s/conf" $environment $version) }}
     mapsApiKey: "{{.Data.mapsApiKey}}",
+    errorReportingApiKey: "{{.Data.errorReportingApiKey}}",
+    projectGcpId: "{{.Data.projectGcpId}}",
```

Comment:
I need some assistance here: I think we should store the key and the projectId in the vault, but I don't know how to put data there. In addition to that, we need to generate different keys for dev, test, staging, and prod, but I have access only to the dev GCP project.
We need to generate keys for test, staging, and prod, using the following example: https://console.cloud.google.com/apis/credentials/key/bc5b7470-db88-4132-a609-3b16164938df?authuser=1&project=broad-ddp-dev&supportedpurview=project